File: Hw1/hw1.pl
#!/usr/bin/perl

my $argNum = $#ARGV; #initialize number of arguments
for (my $i = 0; $i < $argNum+1;$i+=1) { #run while it is not only the script running
    open(fileName, $ARGV[$i]); #open file
    while($line = <fileName>) { #while file is not empty, run
        print $line; #print the file contents
    }
    close(fileName); #clean up, close file
}

File: Hw1/hw1.py
#!/bin/python
import sys #required library for sys functions
i = 1 #initialize value
if len(sys.argv) > 1: #run only if there are arguments passed in
    while(i < (len(sys.argv))): #run while there are files to parse
        sys.stdout.write(open(sys.argv[i],"r+t").read()) #print out lines
        i+=1 #increment